server/src/config.js

```diff
@@ -13,10 +13,17 @@ let conf = convict({
     env: "PORT",
     arg: "port"
   },
+  host: {
+    doc: "The server host",
+    format: String,
+    default: "localhost",
+    env: "HOST",
+    arg: "host"
+  },
   contentPort: {
```

Comment:
What does host header matching look like in express? I think the nginx configuration is going to be pretty much identical, and the port seems like clearer separation to me, but I don't care too much.